docs/UsersGuide/source/ConfigParameters.inc

```diff
@@ -233,106 +668,347 @@ Set parameters associated with the fixed (i.e. static) files.  For the main NOAA
+
+..
+   COMMENT: Does it need to be O? can it be X? or some other variable/letter that doesn't look like the number zero? 
+
+``ISEED_SPPT``: (Default: "1") 
+   Seed for setting the SPPT random number sequence.
+
+``SPPT_MAG``: (Default: "0.7")
+   Amplitude of random patterns. Corresponds to the variable ``sppt`` in ``input.nml``.
+
+``SPPT_LOGIT``: (Default: "TRUE")
+   Limits the SPPT perturbations to between 0 and 2. Should be "TRUE"; otherwise the model will crash.
+
+``SPPT_LSCALE``: (Default: "150000")
+   Decorrelation spatial scales in meters. 
+
+``SPPT_TSCALE``: (Default: "21600") 
+   Decorrelation timescale in seconds. Corresponds to the variable ``sppt_tau`` in ``input.nml``.
+   
+``SPPT_INT``: (Default: "3600") 
+   Interval in seconds to update random pattern (optional parameter). Perturbations still get applied at every time-step. Corresponds to the variable ``spptint`` in ``input.nml``.
+
+``SPPT_SFCLIMIT``: (Default: "TRUE")
+   When "TRUE", tapers the SPPT perturbations to zero at the model’s lowest level, which helps reduce model crashes. 
+
+``USE_ZMTNBLCK``: (Default: "FALSE")
+   When "TRUE", do not apply perturbations below the dividing streamline that is diagnosed by the gravity wave drag, mountain blocking scheme
+
+Stochastic Kinetic Energy Backscatter (SKEB) Parameters
+----------------------------------------------------------
+
+``DO_SKEB``: (Default: "FALSE")
+   Flag to turn Stochastic Kinetic Energy Backscatter (SKEB) on or off. SKEB adds wind perturbations to the model state. Perturbations are random in space/time, but amplitude is determined by a smoothed dissipation estimate provided by the :term:`dynamical core`. SKEB addresses errors in the dynamics more active in the mid-latitudes.
+
+``ISEED_SKEB``: (Default: "3")
+   Seed for setting the SHUM random number sequence.
+
+``SKEB_MAG``: (Default: "0.5") 
+   Amplitude of random patterns. Corresponds to the variable ``skeb`` in ``input.nml``.
+
+``SKEB_LSCALE``: (Default: "150000")
+   Decorrelation spatial scale in meters. 
+
+``SKEB_TSCALE``: (Default: "21600")
+   Decorrelation timescale in seconds. Corresponds to the variable ``skeb_tau`` in ``input.nml``.
+
+``SKEB_INT``: (Default: "3600")
+   Interval in seconds to update random pattern (optional). Perturbations still get applied every time-step. Corresponds to the variable ``skebint`` in ``input.nml``.
+
+``SKEBNORM``: (Default: "1")
+   Patterns:
+      * 0-random pattern is stream function
+      * 1-pattern is K.E. norm
+      * 2-pattern is vorticity
+
+..
+   COMMENT: Stochastic physics docs say the default is zero... Have the numbers changed? Or did the default change?
```

Comment:
This is confusing.  In the model, the default is 0, but in the workflow the default is 1.  
I probably should change the default in the model to also be 1.